### ISSUE TITLE
Build failing because Timestamp.proto file not found

### ISSUE THREAD

**Issue by nkashy1** [doc-ai/tensorio-models#55]
Getting the following error on both `make` and `make docker-build`:
```
cd api && protoc -I . repository.proto --grpc-gateway_out=logtostderr=true:. --proto_path=/go/src --proto_path=/go/pkg/mod --proto_path="/go/pkg/mod/github.com/grpc-ecosystem/grpc-gateway@v1.8.5/third_party/googleapis"
google/protobuf/Timestamp.proto: File not found.
repository.proto: Import "google/protobuf/Timestamp.proto" was not found or had errors.
repository.proto:152:5: "google.protobuf.Timestamp" is not defined.
Makefile:22: recipe for target 'api/repository.pb.gw.go' failed
make: *** [api/repository.pb.gw.go] Error 1
```

**Comment by nkashy1** [comment]
Tried with `protoc` versions `3.6.0` as well as `3.7.1`. Failed.

**Comment by fkautz** [comment]
Fixed with #56
